### PR TITLE
Fix cross-link and format of Chainer at a glance documentation

### PR DESCRIPTION
**Pull request by toslunar**

### CODE REVIEW

**Comment by toslunar** [comment]
Jenkins, test this please.

**Comment by kmaehashi** [comment]
Jenkins, test this please.

File: docs/source/glance.rst [hunk 1]
```diff
@@ -220,120 +220,120 @@ Model
...
 Save two plot images to the result directory.
 
 .. code-block:: python
 
    if extensions.PlotReport.available():
        trainer.extend(
-   	extensions.PlotReport(['main/loss', 'validation/main/loss'],
-   			      'epoch', file_name='loss.png'))
+       extensions.PlotReport(['main/loss', 'validation/main/loss'],
+                     'epoch', file_name='loss.png'))
```
Comment (kmaehashi) [inline docs/source/glance.rst]: Indentation seems incorrect.

File: docs/source/glance.rst [hunk 2]
```diff
@@ -122,25 +122,25 @@ Here's the whole picture of the code:
...
    predict = predict[0][0]
-   
+
    if predict >= 0:
       print('Predicted Poisonous, Actual ' + ['Edible', 'Poisonous'][t[0]])
    else:
       print('Predicted Edible, Actual ' + ['Edible', 'Poisonous'][t[0]])
-   
+
 .. testoutput::
```
Comment (kmaehashi) [inline docs/source/glance.rst]: Strictly speaking it's not related to this PR, but could you add `:hide:` here? It looks odd.

**Comment by toslunar** [comment]
Jenkins, test this please.

**Comment by kmaehashi** [comment]
LGTM! Test failures are not related to this PR.

**Comment by toslunar** [comment]
I squashed the commits.

**Comment by kmaehashi** [comment]
Jenkins, test this please.

**Pull request merged**